CONTRIBUTING.md

```diff
@@ -64,20 +76,55 @@ discuss pending contributions in order to find a resolution. It is expected that
 small minority of issues be brought to the WG for resolution and that discussion and
 compromise among collaborators be the default resolution mechanism.
 
-# Becoming a Member and Collaborator
+Collaborators are expected to follow this policy and continue to send pull requests, go through proper review, and have other collaborators merge their pull requests.
```

Comment:
Not sure about that part. This seems to talk only about "code-like" participation. I'm not sure obligating people to send PRs, and make reviews and be merged be the proper metrics to get the measure of a collaborator's implication.

Reply:
Yeah sorry, still have trouble expressing things. What I meant is that I'm afraid that by phrasing it like that, people will *think* we are talking about code-like participation only. In the end, the nearly only way of participating on GitHub is via comments and PRs and such, but that doesn't mean we have to phrase it like that.
That's just my two cents anyway ^^

Reply:
+1 to both observations. PRs and reviews is in my opinion a reliable structure for constructive contributions to a project, for both non-code and code. I also see the merit in improving our phrasing to avoid misperceptions. We need to be especially aware of discouraging non-code contributions.

Stay tuned for changes here! 📻

Reply:
+1 to both observations. PRs and reviews is in my opinion a reliable structure for constructive contributions to a project, for both non-code and code. I also see the merit in improving our phrasing to avoid misperceptions. We need to be especially aware of discouraging non-code contributions.

Stay tuned for changes here! 📻